common/beerocks/bwl/nl80211/mon_wlan_hal_nl80211.cpp

```diff
@@ -385,6 +385,13 @@ bool mon_wlan_hal_nl80211::sta_link_measurements_11k_request(const std::string &
     return true;
 }
 
+bool mon_wlan_hal_nl80211::channel_scan_trigger(int dwell_time_msec,
+                                                const std::vector<unsigned int> &channel_pool)
+{
+    LOG(TRACE) << __func__;
```

Comment:
add  " - NOT IMPLEMENTED!";

Reply:
Changed, but If i'm not mistaken, all other "not implemented" methods still return true